### ISSUE TITLE
 Most up-to-date way to use Unity?

### ISSUE THREAD

**Issue by lockshaw** [flexflow/FlexFlow#793]
Moved from #579

1. > @jaywonchung
   > It's been some time since Unity was published in OSDI and active development bas been happening in the master branch.
   > At this point, what would be the best way to optimize distributed (multi-node multi-GPU) DNN training with Unity?
   > Is it still the `osdi2022ae` branch, or perhaps is the default optimization method in the master branch Unity?
   > Have the examples been ported to the master branch?
   > 
   > Thanks a lot.
   
2. > @lockshaw 
   > @jaywonchung Thanks for your interest!
   > 
   > All active development is occurring on the `master` branch, so in general that is be the way to go. In the short term there are a few open issues around the search/optimization code on master (as we refactor some of the codebase and development practices to be better for long-term development). We're working on resolving them but for now you might encounter some issues (you can also try jumping over to the `osdi2022ae` branch to see if they also occur there--there may be some places that search/optimization bugs have cropped up on `master` that aren't present in `osdi2022ae`, though in general the code on `master` is in a much better state). 
   > 
   > TLDR: run on `master`, but you may hit some bugs at the moment. If you do, open an issue here and we'll get them fixed asap. :slightly_smiling_face:
